templates/usr/local/sbin/secure_owncloud.sh.j2

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# This script sets the recommended modes and ownership for your ownCloud 
+# directories and files
+# https://doc.owncloud.org/server/9.0/admin_manual/installation/installation_wizard.html#strong-perms-label
+
+# {{ ansible_managed }}
+
```

Comment:
Agreed. Use of the long option names in a code that is non-interactive is a good idea, it improves readability.